config/contracts.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "address": "0x4c00a64addb992a352e5147b791b310b17432a2b"
```

Comment:
How can it support multiple chains?
 It should be something like this:

`{
contract: 0x0a,
chainId: 123,
rpc: https://chain-rpc-url
}`